### ISSUE TITLE
Makecode cannot import package

### ISSUE THREAD

**Issue by CaptainCredible** [MKleinSB/MCP23017#1]
When pasting this packages github adress into makecode  for microbit, the online editor is unable to identify and load the package.

**Comment by MKleinSB** [comment]
Yes, the package is not yet finished for direct loading in makecode. It´s only for use with customblocks.
Meanwhile follow my (german) instructions at hackster:
[](https://www.hackster.io/MKlein/16-digitale-ein-ausgange-am-calliope-mini-custom-blocks-cd1e9e)

**Comment by MKleinSB** [comment]
https://www.hackster.io/MKlein/16-digitale-ein-ausgange-am-calliope-mini-custom-blocks-cd1e9e

**Comment by MKleinSB** [comment]
Or without breadboard
https://www.hackster.io/MKlein/calliope-mini-i2c-io-expander-ffda4d

**Comment by CaptainCredible** [comment]
I made one for importing in the browser, with a few extra functions that I needed: 
https://github.com/CaptainCredible/pxt-MCP23017

**Comment by MKleinSB** [comment]
Looks great! I will add the link to my hackster projects! Thanks!
